nano/node/node.hpp

```diff
@@ -154,6 +154,9 @@ class active_transactions
 	// clang-format on
 	void request_loop ();
 	void request_confirm (std::unique_lock<std::mutex> &);
+	void confirm_frontiers (nano::transaction const &);
+	nano::account last_frontier_account{ 0 };
```

Comment:
This should probably be next_frontier_account since we start at 0 and check the one this points to.

Reply:
Right